specification/columns/capacityreservationstatus.md

```diff
@@ -0,0 +1,42 @@
+# Capacity Reservation Status
+
+Capacity Reservation Status indicates whether the charge represents either the consumption of the [*capacity reservation*](#glossary:capacity-reservation) identified in the CapacityReservationId column or when the *capacity reservation* is unused.
+
+The CapacityReservationStatus column adheres to the following requirements:
+ * CapacityReservationStatus MUST be present in the billing data when the provider supports *capacity reservations* and MUST be of type String.
+ * CapacityReservationStatus MUST be null when CapacityReservationId is null and SHOULD NOT be null when CapacityReservationId is not null.
```

Comment:
```suggestion
 * CapacityReservationStatus MUST be null when CapacityReservationId is null 
 * CapacityReservationStatus MUST NOT be null when CapacityReservationId is not null and [Charge Category](#chargecategory) is "Usage".
```

This is how CommitmentDiscountStatus requirement is specified

> MUST be null when [CommitmentDiscountId](#commitmentdiscountid) is null, and MUST NOT be null when CommitmentDiscountId is not null and [Charge Category](#chargecategory) is "Usage".

Considering, we wish to allow nulls for Purchase records, this should work CapacityReservationStatus, right?

Reply:
@gparker-at-sf I approve Irena's suggestion. 

I wish I wouldve thought of it earlier because it actually is consistent with Commitment purchases where we don't require labeling used vs. unused when you purchase a SP or RI. This is effectively the same scenario but the "purchase" of the ODCR is 0 cost.